Source/OIDAuthorizationRequest.m

```diff
@@ -105,6 +105,20 @@ - (instancetype)init
                    additionalParameters:)
     )
 
+/*! @brief Check if the response type is one AppAuth supports
+    @remarks AppAuth only supports the `code` and `code id_token` response types.
+    @see https://github.com/openid/AppAuth-iOS/issues/98
+    @see https://github.com/openid/AppAuth-iOS/issues/292
+ */
++ (BOOL)isSupportedResponseType:(NSString *)responseType
+{
+  NSString *codeIdToken = [@[OIDResponseTypeCode, OIDResponseTypeIDToken]
```

Comment:
Should we accept `id_token code` as well? I don't know if the order matters, is it documented?

Reply:
Good question. It looks like https://tools.ietf.org/html/rfc6749#section-3.1.1 applies, which says:

>  Extension response types MAY contain a space-delimited (%x20) list of values, where the order of values does not matter (e.g., response type "a b" is the same as "b a").

So yes, we probably should cope with that.

Reply:
I've made this change.

Just to be thorough I also checked on whether multiple spaces are allowed between the values, and by my reading the RFC is explicit that only a single space is permitted.